src/modules/roc_packet/target_stdio/roc_packet/ip_str_formatter.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_packet/target_stdio/roc_packet/ip_str_formatter.h
+//! @brief Convert network address to string.
+
+#ifndef ROC_PACKET_IP_STR_FORMATTER_H_
+#define ROC_PACKET_IP_STR_FORMATTER_H_
+
+#include "roc_core/noncopyable.h"
+#include "roc_packet/address.h"
+
+namespace roc {
+namespace packet {
+
+//! Format ip string of network address.
+class ip_str_formatter : public core::NonCopyable<> {
+public:
+    //! Construct.
+    explicit ip_str_formatter(const Address&);
```

Comment:
It would be better to:

* rename this to `ip_to_str`, like we already have `address_to_str`, `errno_to_str`, etc, etc

* reuse `ip_to_str` in `address_to_str`, like we reuse `address_to_str` in `port_to_str`

    https://github.com/roc-project/roc/blob/3fe1421cf6c7f8ee8399e824259a894e2b2e5eda/src/modules/roc_pipeline/target_stdio/roc_pipeline/port_to_str.cpp#L45-L53

Reply:
Reworked